instrumentation/spring/spring-webflux/spring-webflux-5.3/library/src/test/java/io/opentelemetry/instrumentation/spring/webflux/old/v5_3/TestWebfluxSpringBootOldApp.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.webflux.old.v5_3;
```

Comment:
I had to move this to a non-conflicting package so both `@SpringBootApplication` classes wouldn't be seen and conflict